classes/output/cohort_role_assignments_table.php

```diff
@@ -102,11 +102,12 @@ protected function col_cohortname($data) {
             'idnumber' => $data->cohortidnumber,
             'description' => $data->cohortdescription,
             'visible' => $data->cohortvisible,
-            'name' => $data->cohortname
+            'name' => $data->cohortname,
+            'theme' => null
         );
         $context = context_helper::instance_by_id($data->cohortcontextid);
 
-        $exporter = new \tool_lp\external\cohort_summary_exporter($record, array('context' => $context));
+        $exporter = new \core_cohort\external\cohort_summary_exporter($record, array('context' => $context));
```

Comment:
Thanks Ian - from memory - this class was added around Moodle 3.3? - we'd need to bump the requires version in version.php at the same time as using this - do you want to take a look and fix that too?

Reply:
Hi Dan, no problem. I'll take a look now.